build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+buildscript {
+	repositories {
+		mavenCentral()
+		maven("https://maven.fabricmc.net/") { name = "Fabric" }
+		maven("https://jitpack.io/") { name = "Jitpack" }
+		maven("https://storage.googleapis.com/devan-maven/") { name = "HalfOf2" }
+	}
+	dependencies {
+		classpath("com.github.Chocohead:Fabric-Loom:d823377")
+	}
+}
+plugins { java }
+apply { plugin("fabric-loom") }
+base {
+	val realArchivesBaseName: String by project
```

Comment:
there should only be one archives base name

Reply:
realArchivesBaseName grabs the name from the gradle.properties file.

Reply:
At least cal the one in gradle properties a decent name

Reply:
What should it be named?